specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1675 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Chat Service",
+    "description": "Azure Communication Chat Service",
+    "version": "2020-11-01-preview3"
+  },
+  "paths": {
+    "/chat/threads/{chatThreadId}/readReceipts": {
+      "get": {
+        "tags": [
+          "Threads"
+        ],
+        "summary": "Gets chat message read receipts for a thread.",
+        "operationId": "ChatThread_ListChatReadReceipts",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "Thread id to get the chat message read receipts for.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "maxPageSize",
+            "description": "The maximum number of chat message read receipts to be returned per page.",
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "Skips chat message read receipts up to a specified position in response.",
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The action returns the requested `ChatMessageReadReceipt` resources.",
+            "schema": {
+              "$ref": "#/definitions/ChatMessageReadReceiptsCollection"
+            }
+          },
+          "401": {
+            "description": "Unauthorized.",
+            "schema": {
+              "$ref": "#/definitions/Error"
```

Comment:
We should reference the Error definition from the common.json that we checked in.